user-interface/src/main/java/life/qbic/datamanager/views/general/QbicDialog.java

```diff
@@ -0,0 +1,21 @@
+package life.qbic.datamanager.views.general;
+
+import com.vaadin.flow.component.Key;
+import com.vaadin.flow.component.Shortcuts;
+
+/**
+ * A {@link com.vaadin.flow.component.dialog.Dialog} with additional functionality.
+ * <p>
+ *
+ * @see com.vaadin.flow.component.dialog.Dialog
+ * @since 1.4.0
+ */
+public class QbicDialog extends com.vaadin.flow.component.dialog.Dialog {
```

Comment:
NIT: could just use "Dialog" as extension here? I don't see a naming conflict

Reply:
You are right. I originally called the class Dialog as well thus the concrete class name with packages and all.